backend/src/services/auth.service.ts

```diff
@@ -119,6 +106,15 @@ export async function getUserFromToken(token: string): Promise<UserDocument> {
   }
 }
 
+async function vaccAuth(cid: string): Promise<boolean> {
```

Comment:
please expect an object as the parameter as this might require more data in the future and that will reduce potential refactoring needed

example:

```js
// function definition:
function vaccAuth(authData) {
  // ...
}

// function usage:
await vaccAuth({ cid: userFromApi.cid });
```